perfmetrics/scripts/continuous_test/ml_tests_tf/continuous.cfg

```diff
@@ -0,0 +1,2 @@
+build_file: "gcsfuse/perfmetrics/scripts/continuous_test/ml_tests_tf/build.sh"
```

Comment:
instead of calling it ml_tests_tf. - can you make a directory ml_tests/tf/your scripts

Reply:
Done